$NetBSD: patch-af,v 1.1 2011/01/15 17:46:01 tron Exp $

Remove unused custom function "getline" which conflicts with getline(3)
which is part of the POSIX standard.

--- lib/stdio/fgetline.c.orig	2000-12-03 11:19:29.000000000 +0000
+++ lib/stdio/fgetline.c	2011-01-15 17:42:18.000000000 +0000
@@ -62,11 +62,3 @@
 
 	return (bp - buf);
 }
-
-EXPORT int
-getline(buf, len)
-	char	*buf;
-	int	len;
-{
-	return (fgetline(stdin, buf, len));
-}
